### PR TITLE
PHPCS improvements & Unit tests for PaymentData and Paymentorder classes

### PR DESCRIPTION
**Pull request by shahriar-vdc**

### CODE REVIEW

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- Test/Unit/PaymentorderTest.php  1
- Test/Unit/PaymentDataTest.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-magento2-checkout/pullRequest?prid=5821673&bid=19059223)

File: Test/Unit/PaymentorderTest.php [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+use SwedbankPay\Api\Service\Paymentorder\Resource\PaymentorderSwish;
+use SwedbankPay\Checkout\Helper\Config as PaymentMenuConfig;
+use SwedbankPay\Checkout\Helper\Paymentorder;
+use SwedbankPay\Checkout\Model\QuoteFactory;
+use SwedbankPay\Checkout\Model\ResourceModel\QuoteRepository;
+use SwedbankPay\Core\Helper\Config as ClientConfig;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+class PaymentorderTest extends TestCase
```
Comment (payex-dev) [inline Test/Unit/PaymentorderTest.php:35]: Codacy found an issue: [The class PaymentorderTest has a coupling between objects value of 25. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-magento2-checkout/pullRequest?prid=5821673)

Comment (asbjornu) [reply Test/Unit/PaymentorderTest.php:35]: Perhaps add a comment to ignore whatever rule Codacy thinks is broken here?

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Test/Unit/PaymentorderTest.php  1
- Test/Unit/PaymentDataTest.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-magento2-checkout/pullRequest?prid=5821673&bid=19059223)

**Pull request merged**